content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,136 @@
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
 weight: 110
 aliases:
     - "/concepts/transports/webrtc"
 ---
 
-Coming soon!
+## What is WebRTC?
+
+[WebRTC (Web Real-Time Communications)](https://webrtc.org/) is a framework for real-time
+communication and in libp2p is used to establish browser-to-server and browser-to-browser
+connections between applications.
+
+WebRTC was originally designed to make audio, video, and data
+communication between browsers user-friendly and easy to implement.
+It was first developed by [Global IP Solutions (or GIPS)](http://www.gipscorp.com/).
+In 2011, GIPS was acquired by Google where the [W3C](https://www.w3.org/) started to work
+on a standard for WebRTC.
+
+It serves as a good choice for applications that need built-in support
+for media communication and do not have specific requirements for the underlying
+transport protocol.
+
+## WebRTC in libp2p
+
+In libp2p, WebRTC is used as a transport protocol to connect from browsers to other nodes.
+However, libp2p does not make use of any of WebRTC's multimedia features.
+The features employed in libp2p are:
+
+### Key features
+
+- Peer connections: WebRTC enables
+  [direct peer-to-peer connections](https://webrtc.org/getting-started/peer-connections)
+  between browsers and other nodes.
+
+- Data channels: WebRTC provides peer-to-peer [data channels](https://developer.mozilla.org/en-US/docs/Games/Techniques/WebRTC_data_channels),
+  which works on
+  [SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol) and use
+  [SDP (Session Description Protocol)](https://en.wikipedia.org/wiki/Session_Description_Protocol) to negotiate the parameters
+  of the data channel, such as the type of data that will be sent, the codecs that will be used to encode the data, and
+  other properties.
+
+  A WebRTC data channel allows applications to send a text or binary data over an active connection
+  to a peer. This means libp2p can utilize data channels as a transport to send raw data to peers and
+  enables applications to build anything they like.
+
+- [NAT traversal](../nat/overview): WebRTC includes mechanisms (like
+  [ICE](https://datatracker.ietf.org/doc/rfc5245/)) to connect to nodes that run behind
+  NATs and firewalls. In non-decentralized WebRTC, this can be facilitated by a
+  [TURN server.](https://webrtc.org/getting-started/turn-server),
+  but other signaling channels, such as WebSocket running on a central server, can also be used.
+  Using a custom [signaling protocol](https://en.wikipedia.org/wiki/Signaling_protocol) or a
+  different signaling service is also possible. Overall, this allows for faster and more efficient communication.
```

Comment:
What allows for faster and more efficient communication? The alternative signaling services? Note that neither faster nor more efficient was the motivation for libp2p to not use TURN. The motivation was (a) not depending on central (TURN) servers and (b) leveraging existing libp2p protocols (circuit relay v2). I suggest removing the last sentence.